object_list_1:
  - name: biscuits
    group: green
  - name: soap
    group: green
  - name: soap2
    group: red
